kernels/armsve/3/bli_gemm_armsve256_asm_d8x8.c

```diff
@@ -70,35 +74,35 @@ void bli_dgemm_armsve256_asm_8x8
 
 __asm__ volatile
 (
-"                                            \n\t" 
-" ldr x0,%[aaddr]                            \n\t" // Load address of A 
-" ldr x1,%[baddr]                            \n\t" // Load address of B
-" ldr x2,%[caddr]                            \n\t" // Load address of C
-"                                            \n\t"
-" ldr x3,%[a_next]                           \n\t" // Move pointer
-" ldr x4,%[b_next]                           \n\t" // Move pointer
-"                                            \n\t"
-" ldr x5,%[k_iter]                           \n\t" // Init guard (k_iter)
-" ldr x6,%[k_left]                           \n\t" // Init guard (k_iter)
-"                                            \n\t" 
-" ldr x7,%[alpha]                            \n\t" // Alpha address      
-" ldr x8,%[beta]                             \n\t" // Beta address      
-"                                            \n\t" 
-" ldr x9,%[cs_c]                             \n\t" // Load cs_c
-" lsl x10,x9,#3                              \n\t" // cs_c * sizeof(double)
-"                                            \n\t"
-" ldr x13,%[rs_c]                            \n\t" // Load rs_c.
-" lsl x14,x13,#3                             \n\t" // rs_c * sizeof(double). 
-"                                            \n\t"
-" add x20,x2,x10                             \n\t" //Load address Column 1 of C
+// "                                            \n\t" 
+// " ldr x0,%[aaddr]                            \n\t" // Load address of A 
+// " ldr x1,%[baddr]                            \n\t" // Load address of B
+// " ldr x2,%[caddr]                            \n\t" // Load address of C
+// "                                            \n\t"
+// " ldr x3,%[a_next]                           \n\t" // Move pointer
+// " ldr x4,%[b_next]                           \n\t" // Move pointer
+// "                                            \n\t"
+// " ldr x5,%[k_iter]                           \n\t" // Init guard (k_iter)
+// " ldr x6,%[k_left]                           \n\t" // Init guard (k_iter)
+// "                                            \n\t" 
+// " ldr x7,%[alpha]                            \n\t" // Alpha address      
+// " ldr x8,%[beta]                             \n\t" // Beta address      
+// "                                            \n\t" 
+// " ldr x9,%[cs_c]                             \n\t" // Load cs_c
+// " ldr x13,%[rs_c]                            \n\t" // Load rs_c.
+"                                            \n\t"
+" lsl x10,%9,#3                              \n\t" // cs_c * sizeof(double)
+" lsl x14,%10,#3                             \n\t" // rs_c * sizeof(double). 
+"                                            \n\t"
+" add x20,%2,x10                             \n\t" //Load address Column 1 of C
```

Comment:
I think I'm going to push a patch to use asm names (e.g. %[c]) for whoever ends up maintaining this in the future.